OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -162,6 +162,23 @@ public GameInfoStatsLogic(Widget widget, ModData modData, World world, OrderMana
 					muteCheckbox.IsVisible = () => !pp.IsBot && client.State != Session.ClientState.Disconnected && pp.ClientIndex != orderManager.LocalClient?.Index;
 					muteCheckbox.GetTooltipText = () => muteCheckbox.IsChecked() ? unmuteTooltip : muteTooltip;
 
+					var kickButton = item.Get<ButtonWidget>("KICK");
+					kickButton.IsVisible = () => Game.IsHost && client.Index != orderManager.LocalClient.Index && client.State != Session.ClientState.Disconnected && pp.WinState == WinState.Lost && !pp.IsBot;
+					kickButton.OnClick = () =>
```

Comment:
Copy paste the `kickButton.OnClick` from line 215 to here. The way we define prompts has changed in #20142